content/_includes/CORETopMenu.md

```diff
@@ -53,21 +53,21 @@ The alert system is configured in [System > Alert Settings]({{< relref "AlertSet
 
 The <span class="iconify" data-icon="fa6-solid:gear"></span> (gear icon) contains links to various system specific options.
 
-<span class="iconify" data-icon="ic:baseline-dialpad"></span> **Change Password** is a shortcut for changing the administor (*root*) account password.
+<span class="iconify" data-icon="ic:baseline-dialpad"></span> **Change Password** is a shortcut for changing the administrator (*root*) account password.
 This password is required to log in to the TrueNAS web interface.
 Please back up or otherwise memorize the updated password when changing it.
 
 <span class="iconify" data-icon="ic:outline-settings-applications"></span> **Preferences** contains theme and other visual options for the web interface:
 
 | Option | Description |
 |--------|-------------|
-| Choose Theme | Choose a preferred theme. There are several built-in themes designed for light and dark modes and high contrast viewing of the web interface. |
-| Prefer buttons with icons only | Preserve screen space with icons and tooltips instead of text labels. |
-| Enable Password Toggle | When set, an eye icon appears next to password fields. Clicking the icon reveals the password. |
-| Reset Table Columns to Default | Reset all tables to display default columns. |
-| Retro Logo | Revert branding back to FreeNAS. |
-| Reset All Preferences to Default | Reset all user preferences to their default values. (Custom themes are preserved) |
-| Update Preferences | Save changes to the **General Preferences**. |
+| **Choose Theme** | Choose a preferred theme. There are several built-in themes designed for light and dark modes and high contrast viewing of the web interface. |
+| **Prefer buttons with icons only** | Preserve screen space with icons and tooltips instead of text labels. |
+| **Enable Password Toggle** | When set, an eye icon appears next to password fields. Clicking the icon reveals the password. |
+| **Reset Table Columns to Default** | Reset all tables to display default columns. |
+| **Retro Logo** | Revert branding back to FreeNAS. |
+| **Reset All Preferences to Default** | Reset all user preferences to their default values. (Custom themes are preserved) |
+| **Update Preferences** | Save changes to the **General Preferences**. |
 
```

Comment:
Select not choose even if the field name includes the word choose. The descriptions need modifications as described above in previous comments. Are they selecting from a dropdown? from button options? Are some of these checkboxes? Standardizing the description language helps with consistent presentation throughout all documentation. Most of this is covered in the Simple Rules for Writing Good articles in Confluence. :-)